### PR TITLE
Add job definition for opentelekomcloud provider

### PR DESCRIPTION
**Pull request by liu-sheng**
For #34

### CODE REVIEW

**Comment by liu-sheng** [comment]
WIP: still need to add a OS_SECRET_KEY environment variable for running acceptance tests, but we cannot know it for now.

**Comment by liu-sheng** [comment]
now, this is ready to review

File: playbooks/terraform-provider-opentelekomcloud-acceptance-test-telekom/run.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          export OS_NETWORK_NAME="$_NET_PREFIX-net"
+          export OS_NETWORK_ID="$(openstack network show $OS_NETWORK_NAME -f value -c id)"
+          if [ -z "$OS_NETWORK_ID" ]; then
+              export OS_NETWORK_ID="$(openstack network create $OS_NETWORK_NAME -f value -c id)"
+              openstack subnet create --network $OS_NETWORK_ID --subnet-range 10.0.20.0/24 $_NET_PREFIX-subnet
+          fi
+
+          export OS_SWIFT_ENVIRONMENT=1
+
+          cd $GOPATH/src/github.com/gator1/terraform-provider-opentelekomcloud/
```
Comment (kiwik) [inline playbooks/terraform-provider-opentelekomcloud-acceptance-test-telekom/run.yaml]: Looks this line is not necessary.

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -176,3 +176,13 @@
     run: playbooks/terraform-provider-flexibleengine-acceptance-test-orange/run.yaml
     secrets:
       - orange_credentials
+
+# terraform-provider-opentelekomcloud acceptance tests with Telekom cloud
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: s/Telekom cloud/Open Telekom Cloud/

File: zuul.d/jobs.yaml [hunk 3]
```diff
@@ -176,3 +176,13 @@
...
+- job:
+    name: terraform-provider-opentelekomcloud-acceptance-test-telekom
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: Please use fullname for the last word.
s/telekom/opentelekomcloud/

File: zuul.d/jobs.yaml [hunk 4]
```diff
@@ -176,3 +176,13 @@
...
+    parent: golang-test
+    description: |
+      Run acceptance tests of terraform-provider-opentelekomcloud repo against telekom cloud
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: ditto

File: zuul.d/jobs.yaml [hunk 5]
```diff
@@ -176,3 +176,13 @@
...
+    run: playbooks/terraform-provider-opentelekomcloud-acceptance-test-telekom/run.yaml
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: ditto

File: zuul.d/jobs.yaml [hunk 6]
```diff
@@ -176,3 +176,13 @@
...
+    secrets:
+      - telekom_credentials
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: ditto

File: zuul.d/secrets.yaml [hunk 7]
```diff
@@ -145,3 +145,97 @@
           gezU1HyiWELy5ZrG/E1Koy1GYKDNCzfzIbaIiLQq+777iZE0aS4sMsU+bt8knB6fL3wk7
           eP1vIoCmUuApZl4TiXYXB71xp3PVQe3iWEhc3d0R2rUtsLjl+2O4rJFW2nzUL6aZFgAgG
           4qOD83LuryOVSQAMfDXzQOTxUpJtwf65ryKY2bELo0ceR4eoUsiqB+C31IMukk=
+
+- secret:
+    name: telekom_credentials
```
Comment (kiwik) [inline zuul.d/secrets.yaml]: ditto

File: playbooks/terraform-provider-opentelekomcloud-acceptance-test-telekom/run.yaml (hunk 1 above)
Comment (liu-sheng) [reply playbooks/terraform-provider-opentelekomcloud-acceptance-test-telekom/run.yaml]: yes, it is, done

File: zuul.d/jobs.yaml (hunk 3 above)
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/jobs.yaml (hunk 2 above)
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/jobs.yaml (hunk 4 above)
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/jobs.yaml (hunk 6 above)
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/secrets.yaml (hunk 7 above)
Comment (liu-sheng) [reply zuul.d/secrets.yaml]: done

**Comment by kiwik** [comment]
2 Approved, let's merged this patch.

**Pull request merged**